source/test/main.cc

```diff
@@ -0,0 +1,137 @@
+#include "host/System.hh"
+
+#include <test/TestDirector.hh>
```

Comment:
Pedantic, but the convention so far is for headers in the same directory come first and use quotes, and headers in other folders come after and use angled braces
```suggestion
#include "test/TestDirector.hh"

#include <host/System.hh>
```